app-text/xapers/xapers-0.8.2-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Personal document indexing system"
+HOMEPAGE="https://finestructure.net/xapers/"
+SRC_URI="https://finestructure.net/xapers/releases/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="app-text/poppler[utils]
+	dev-libs/xapian-bindings[python,${PYTHON_USEDEP}]
+	dev-python/pybtex[${PYTHON_USEDEP}]
+	dev-python/pycurl[${PYTHON_USEDEP}]
+	dev-python/urwid[${PYTHON_USEDEP}]
+	x11-misc/xclip
+	x11-misc/xdg-utils"
+
+DEPEND="${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+src_tests() {
+	./test.sh
```

Comment:
Append with `|| die`.

Truthfully, I don't think this even does anything at the moment.
```
>>> Source compiled.
>>> Test phase: app-text/xapers-0.8.2-r1
>>> Completed testing app-text/xapers-0.8.2-r1
```

Make sure to test the ebuild after adding die there.

Reply:
Quite a few bugs in my code there, including mis-naming `src_test` as `src_tests` and calling completely the wrong file. Have renamed function, and called correct test script. Tests all pass (bar one known broken, doesn't cause failure) on my end using the ebuild now.